mLRS/Common/frames.h

```diff
@@ -14,6 +14,7 @@
 #include "frame_types.h"
 
 
+extern tGlobalConfig Config;
```

Comment:
I'm curious why we didn't need this before.

Reply:
ha. These are not needed at all since these are .h files. I just started to add them even if nfc to make the dependencies more clear.